tools/keytools/sign.c

```diff
@@ -127,6 +130,7 @@
 
 #define HDR_IMG_TYPE_AUTH_MASK    0xFF00
 #define HDR_IMG_TYPE_AUTH_NONE    0xFF00
+/*
```

Comment:
Why are these commented out?

Reply:
these are re definitions from wolfboot.h, I'll just delete them since I'm not including it